ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/discovery/PeerDiscoveryAgentTest.java

```diff
@@ -898,6 +901,23 @@ public void assertHostCorrectlyRevertsOnIgnoredPacketFrom() {
     assertThat(PeerDiscoveryAgent.deriveHost(source, mockWellFormed)).isEqualTo(routableHost);
   }
 
+  @Test
+  void testFromEnodeWithDiscoveryDisabled() throws UnknownHostException {
+    EnodeURL enodeWithNoDiscovery = mock(EnodeURL.class);
+    when(enodeWithNoDiscovery.getDiscoveryPort()).thenReturn(Optional.empty());
+    when(enodeWithNoDiscovery.getListeningPort()).thenReturn(Optional.of(8545));
+
+    when(enodeWithNoDiscovery.getIp()).thenReturn(InetAddress.getLoopbackAddress());
+
+    Endpoint result = Endpoint.fromEnode(enodeWithNoDiscovery);
+
+    assertEquals("127.0.0.1", result.getHost());
+
+    assertEquals(30303, result.getUdpPort());
```

Comment:
Should use the default port constant here so that the value is only in one place

Reply:
done